docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Undertow.adoc

```diff
@@ -249,6 +249,49 @@ be matched
 serve up requests that do not match anything.
 |=======================================================================
 
+==== Console Access Logging
+
+Each host allows for access logging to the console which writes structured
+data in JSON format. This only writes to `stdout` and is a single line of
+JSON structured data.
+
+The `attributes` management model attribute is used to determine which exchange attributes should be logged. This is
+similar to the `pattern` used for traditional access logging. The main difference being since the data is structured
+the ability to use defined keys is essential.
+
```

Comment:
There should also be a paragraph like...

The `metadata` management model attribute can be used to add extra metadata to each log record. The value of the attribute is an arbitrary set of key/value pairs. The values can include management model expressions, which must be resolvable when the console access log service is started. The value is resolved once per start or reload of the server.